src/Apps/Order/Routes/__tests__/Accept.test.tsx

```diff
@@ -1,43 +1,28 @@
-import { Button } from "@artsy/palette"
 import {
   Buyer,
   OfferOrderWithShippingDetails,
   Offers,
   OfferWithTotals,
 } from "Apps/__tests__/Fixtures/Order"
-import { ArtworkSummaryItemFragmentContainer } from "Apps/Order/Components/ArtworkSummaryItem"
-import { CreditCardSummaryItemFragmentContainer } from "Apps/Order/Components/CreditCardSummaryItem"
-import { OrderStepper } from "Apps/Order/Components/OrderStepper"
-import { ShippingSummaryItemFragmentContainer } from "Apps/Order/Components/ShippingSummaryItem"
-import { TransactionDetailsSummaryItemFragmentContainer } from "Apps/Order/Components/TransactionDetailsSummaryItem"
-import { ConnectedModalDialog } from "Apps/Order/Dialogs"
 import { trackPageView } from "Apps/Order/Utils/trackPageView"
-import { ModalButton, ModalDialog } from "Components/Modal/ModalDialog"
-import { Stepper } from "Components/v2"
-import { CountdownTimer } from "Components/v2/CountdownTimer"
-import { MockBoot } from "DevTools"
-import { mount } from "enzyme"
+import { createTestEnv } from "DevTools/createTestEnv"
 import moment from "moment"
-import React from "react"
-import { commitMutation as _commitMutation } from "react-relay"
-import { flushPromiseQueue } from "Utils/flushPromiseQueue"
+import { graphql } from "react-relay"
 import {
   acceptOfferFailed,
   AcceptOfferPaymentFailed,
   acceptOfferSuccess,
 } from "../__fixtures__/MutationResults"
-import { AcceptFragmentContainer as AcceptRoute } from "../Accept"
-
-const commitMutation = _commitMutation as jest.Mock<any>
+import { AcceptFragmentContainer } from "../Accept"
+import { OrderAppTestPage } from "./Utils/OrderAppTestPage"
```

Comment:
If `OrderAppTestPage` implements a requirements defined in `createTestEnv` can we add some docs to the `DevTools` folder?